server/routes/request.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { getRequests, postRequest, updateRequest } = require("../controllers/request");
+
+router.route("/").get(protect, getRequests);
+router.route("/new").post(protect, postRequest);
```

Comment:
It's a common convention to use "/" for this route (HTTP methods are sufficient to differentiate `get/create`).